packages/app/src/specs/CreateSpecModal.vue

```diff
@@ -6,7 +6,7 @@
     :title="title"
     :model-value="show"
     data-testid="create-spec-modal"
-    @update:model-value="$emit('close')"
+    @update:model-value="close"
```

Comment:
The standard modal has a help link by default and it seems to take you to the homepage of the docs. Is there anyway to disable that help link or send it somewhere useful?

Reply:
I believe its supposed to be a general "catch-all" link, not sure about disabling but as it stands all generators have that for now so I won't make a change in this PR.

Reply:
@ImCesar It's a huge pain to control outside of the modal. That's what `:title` is for. We can easily add a link to come back from the child.